pkgs/applications/virtualization/lima/bin.nix

```diff
@@ -55,6 +55,8 @@ stdenvNoCC.mkDerivation {
     mkdir -p $out
     cp -r bin share $out
     chmod +x $out/bin/limactl
+    # patch limactl here separately to generate completion with it
+    autoPatchelf -- $out/bin/limactl
```

Comment:
```suggestion
```

Reply:
Remove this again because if you move the installShellCompletion to the fixup Phase, then manual patching is not necessary anymore.